<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16">
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="maximumSize">
      <size>
       <width>520</width>
       <height>360</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>520</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>497</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(51, 117, 172);
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>ADD BUDGET</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>520</width>
          <height>306</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>486</width>
          <height>318</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-top-right-radius:0px;
border-top-left-radius:0px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-bottom-radius:1px solid rgb(166, 166, 166);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_14">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Categories:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_11">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item alignment="Qt::AlignVCenter">
                <widget class="QComboBox" name="categories_combobox">
                 <property name="minimumSize">
                  <size>
                   <width>118</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>-1</pointsize>
                   <weight>50</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox {
    background-color: #fdfdfd;
    border: 1px solid #ccc;
    border-radius: 6px;
    padding: 4px 6px;
    font: 13px &quot;Segoe UI&quot;;
    combobox-popup: 0;
}

QComboBox:hover {
    border: 1px solid #a5a5a5;
}

QComboBox:focus {
    border: 1px solid #5ab1ef;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #ccc;
    background-color: transparent;
}

QComboBox::down-arrow {
    image: url(:/icon/chevron-down.svg);
    width: 10px;
    height: 10px;
}

QComboBox::down-arrow:disabled {
    image: none;
}
</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Food</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Bills and utilities</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Shopping</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Family</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Vehicle</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Health</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Education</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Entertaiment</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Others</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-bottom-radius:1px solid rgb(211, 211, 211);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QFrame" name="frame_12">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Amount:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_13">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLineEdit" name="amount_line_edit">
                 <property name="minimumSize">
                  <size>
                   <width>355</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 2px solid rgba(0, 0, 0, 0);
border-bottom-color: rgba(46, 82, 101, 200);
color: rgb(0, 0, 0);
padding-bottom: 7px;
border-radius: 0px;
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-bottom-radius:1px solid rgb(211, 211, 211);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_16">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <property name="rightMargin">
                <number>4</number>
               </property>
               <item alignment="Qt::AlignLeft">
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Time:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_15">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item>
                <widget class="QComboBox" name="TimecomboBox">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QComboBox {
    background-color: #fdfdfd;
    border: 1px solid #ccc;
    border-radius: 6px;
    padding: 4px 6px;
    font: 13px &quot;Segoe UI&quot;;
    combobox-popup: 0;
}

QComboBox:hover {
    border: 1px solid #a5a5a5;
}

QComboBox:focus {
    border: 1px solid #5ab1ef;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #ccc;
    background-color: transparent;
}

QComboBox::down-arrow {
    image: url(:/icon/chevron-down.svg);
    width: 10px;
    height: 10px;
}

QComboBox::down-arrow:disabled {
    image: none;
}
</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>This week</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>This month</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>This Year</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_check">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="text_error">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>check</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_19">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>72</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QFrame" name="frame_20">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_19">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="add_btn">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>38</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#add_btn{
	background-color: rgb(9, 101, 200);
	color: rgba(255, 255, 255, 255);
	border-radius: 5px;
}
#add_btn:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(255, 107, 107, 255);
}

#add_btn:hover {
background-color: rgb(7, 64, 157);
}</string>
                 </property>
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_21">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="cancel_btn">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>38</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#cancel_btn{
	background-color: rgb(230, 0, 0);
	color: rgba(255, 255, 255, 255);
	border-radius: 5px;
}
#cancel_btn:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(255, 107, 107, 255);
}

#cancel_btn:hover {
	
	background-color: rgb(140, 0, 0);
}</string>
                 </property>
                 <property name="text">
                  <string>Cancel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
